src/main/frontend/app/containers/BarShowContainer.js

```diff
@@ -0,0 +1,54 @@
+import React, {Component} from 'react';
+import ReviewFormContainer from './ReviewFormContainer'
+
+class BarShowContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bar: {},
+      reviews: []
+    }
+  }
+
+  componentDidMount() {
+    // let selectedBarId = this.props.params.id
+    let pathArray = window.location.pathname.split('/');
+    let barId = pathArray[pathArray.length-1];
+    fetch(`/api/v1/bars/${barId}`)
+    .then(response => {
+      return response.json()
+  })
+    .then(bar => {
+      this.setState( {bar: bar} )
+      this.setState( {reviews: bar.reviews} )
+    })
+  }
+
+  render() {
+    let reviewsArray = this.state.reviews.map(review => {
+      return(
+        <div>
```

Comment:
if we're building a list, we can't have a `<div>` between the `<ul>` and the `<li>`